### DISCUSSION TITLE
Prioritize One OTP Form over the others

### DISCUSSION THREAD

**Discussion by PaulSteindl** [keycloak/keycloak#22477] [Q&A]
What the title says, maybe I need help understanding the documentation. How can I set my Email OTP as the default form? I want the other twos as alternatives, but for some reason, it always chooses the OTP Form. Any advice?

![image](https://github.com/keycloak/keycloak/assets/81384142/a7e7e826-afdf-4370-9f42-ad1797aa2c1e)

**Comment by darius-m** [comment]
It may be that it was saved as the preferred OTP method on a per-account basis. I think each account could choose which option to use (by making it the first option in the credentials list), but the option to update it may be missing from the account interface. To test whether this is the case, you could delete the OTP and re-configure it on an account that has both configured, but the OTP is the first option in the list. Not sure what a non-hacky workaround would be.

**Reply by PaulSteindl** [reply]
Okay, thanks for the input
